### PR TITLE
#5Task. Nadezhda Kovaleva

### PR DESCRIPTION
**Pull request by Nadia-Kovaleva**
https://travis-ci.org/github/Nadia-Kovaleva/js-assignments/builds

### CODE REVIEW

**Comment by FedorovD** [comment]
Please clear PR code not related to Task#5 and click "ready for review" when it will be finished. Thank you!
image

**Comment by Nadia-Kovaleva** [comment]
Task/6 file does not contain any changes now, I returned it to its original form. Now the changes in PR apply only to task/5 file. 
I do not really understand how I can clear PR code not related to Task#5.

File: task/05-regex-tasks.js [hunk 1]
```diff
@@ -91,10 +87,9 @@ function getRegexForIPv4() {
  * @return {RegExp}
  */
 function getRegexForSSN() {
-   throw new Error('Not implemented');
+  return /^(?!(000))\d{3}-(?!00)\d{2}-(?!0000)\d{4}$/;
```
Comment (FedorovD) [inline task/05-regex-tasks.js]: Some special numbers are never allocated:
Numbers with all zeros in any digit group (000-##-####, ###-00-####, ###-##-0000).
**Numbers with 666 or 900-999 (Individual Taxpayer Identification Number) in the first digit group.**

not correct, highlighted rules are not validated

**Comment by FedorovD** [comment]
It seems you're using some formatting tool, it makes a lot of unnecessary changes.
fix `getRegexForSSN` task
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by FedorovD** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**